polars/polars-sql/src/sql_expr.rs

```diff
@@ -99,15 +101,31 @@ impl SqlExprVisitor<'_> {
             idents.len() == 2,
             ComputeError: "compound identifier {:?} is not yet supported", idents,
```

Comment:
Shouldn't this error message change? As we do support "compound identifiers". What don't we support now?